barcode_dc/code/barcode_scan.py

```diff
@@ -101,12 +101,13 @@ def find_scanner(self):
 
                 try:
                     if dev.properties['ID_INPUT_KEYBOARD'] == "1":
-                        _, connection_point = dev.properties['ID_PATH'].split('-usb-')
+                        platform, connection_point = dev.properties['ID_PATH'].split('-usb-')

```

Comment:
If the code is quietly doing string operations on the config (eg split), the config file  needs lots of syntax docs.

Reply:
That's not on the config, that's on the os reported device